### DISCUSSION TITLE
Update for KirisameR's Sierra-Wireless-EM74xx-Series-WWAN-Card-Driver-for-macOS-Catalina

### DISCUSSION THREAD

**Discussion by GeorgettaCZ** [community/community#39234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

Just a quick update to point 1 that I discovered here:
https://github.com/KirisameR/Sierra-Wireless-EM74xx-Series-WWAN-Card-Driver-for-macOS-Catalina

It is not enough just to hover the cursor over the version of the macOS in About this MAC. You need to click on it to get the long macOS number.

Thats all ;o)

Bye! Good luck!

Gerorge

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by GeorgettaCZ** [comment]
Hello,

Thank you, I am fine with what get. I just wanted to point out something that I found important and missing published.

Bye, good luck!
